CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# Release 0.13.0
+- [GH-1539] Stop started workloads in system tests ([#562](https://github.com/broadinstitute/wfl/pull/562))
+- [GH-1563] [GH-1573] TerraDataRepoSource polls row metadata table ([#559](https://github.com/broadinstitute/wfl/pull/559))
+- [GH-1433] Test TDR snapshot creation within temporary dataset ([#561](https://github.com/broadinstitute/wfl/pull/561))
+- [GH-1578] TerraDataRepoSource filters old rows before reducing ([#560](https://github.com/broadinstitute/wfl/pull/560))
+- [GH-1542] TerraExecutor notifies Slack watchers on workflow completion ([#553](https://github.com/broadinstitute/wfl/pull/553))
+
+# Release 0.12.1
```

Comment:
Oh, I guess we didn't have the 0.12.1 changelog in yet:

https://github.com/broadinstitute/wfl/blob/develop/CHANGELOG.md